tests/func/test_repro.py

```diff
@@ -1176,7 +1176,7 @@ def test(self):
 
         self.dvc.remove("imported_file.dvc")
 
-        with StaticFileServer(handler="Content-MD5") as httpd:
+        with StaticFileServer(handler_class=ContentMD5Handler) as httpd:
```

Comment:
```suggestion
        with StaticFileServer(handler=ContentMD5Handler) as httpd:
```
It's up to you though.